src/outliers/visualize_outliers.py

```diff
@@ -1,16 +1,29 @@
-def visualize_outliers(dataframe, columns=None):
+import pandas as pd
+import altair as alt
+
+def visualize_outliers(dataframe, columns=None, type='violin'):
     """
-    A function that plot out the given data with outliers marked.
+    A function that plot the distribution of the give data.
```

Comment:
```suggestion
    A function that plot the distribution of the given data.
```

Reply:
Thank you for all the corrections there are really tooooooo much typo for me